src/pages/create-property/newProperty-page.component.jsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import AddPropertyComponent from '../../components/add-property/AddProperty.component';
```

Comment:
- [ ] I can add a property, however, I cannot do anything with it and this screen shows up on click:


![image](https://user-images.githubusercontent.com/57421823/203672860-2ff87b9b-4a9a-4193-8612-8012bf6e572d.png)

Kindly fix this bug so your app works as intended 👍